src/main/java/movie/storage/controllers/ShoppingCartController.java

```diff
@@ -29,14 +29,14 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @GetMapping("/by-user")
-    public ShoppingCart getByUser(@RequestParam String userEmail) {
-        return shoppingCartService.getByUser(userService.findByEmail(userEmail));
+    public ShoppingCart getByUser(Authentication authenticationUser) {
+        return shoppingCartService.getByUser(userService.findByEmail(authenticationUser.getName()));
     }
 
     @PostMapping("/movie-sessions")
-    public void addMovieSession(@RequestParam String userEmail,
+    public void addMovieSession(Authentication authenticationUser,
                                 @RequestBody MovieSessionRequestDto movieSession) {
```

Comment:
```suggestion
                                @RequestParam Long movieSessionId) {
```
let's pass movieSessionId instead